frontend/services/entities/mdm_apple.ts

```diff
@@ -13,12 +13,43 @@ export interface IVppApp {
   latest_version: string;
   app_store_id: number;
   added: boolean;
+  platform: string; // darwin | ios | ipados
```

Comment:
I think I added `ApplePlatform` to `platform.ts` in this PR that should be merged shortly https://github.com/fleetdm/fleet/pull/20591/files

Reply:
I did as well in the following PR for this work

Reply:
Since that work will need a mapping of platforms to platform display names that PR will do it like this: https://github.com/fleetdm/fleet/pull/20755/files#diff-ba78711f30db3556ff3d05a8cd4e3811b93cbbf11d289f615bf6a75388ffe03c